docs/designers-developers/developers/tutorials/javascript/versions-and-building.md

```diff
@@ -0,0 +1,14 @@
+
+## JavaScript versions and build step
+
+The Gutenberg Handbook shows JavaScript examples in two syntaxes ES5 and ESNext. These are version names for the JavaScript language standard definitions. You may also see elsewhere the versions ES6, or ECMAScript 2015 mentioned. Ses the [ECMAScript](https://en.wikipedia.org/wiki/ECMAScript) Wikipedia article for all the details.
+
+ES5 code is compatible with WordPress's minimum [target for browser compatibility](https://make.wordpress.org/core/2017/04/23/target-browser-coverage/).
```

Comment:
We may want to point to the [browser support handbook](https://make.wordpress.org/core/handbook/best-practices/browser-support/) or [the Gutenberg FAQ](https://wordpress.org/gutenberg/handbook/designers-developers/faq/#what-browsers-will-gutenberg-support) instead, they seem more permanent. I'd say the core handbook is probably best.

Reply:
👍Updated the link.